src/coreclr/jit/llvm.cpp

```diff
@@ -832,6 +832,30 @@ void Llvm::buildAdd(GenTree* node, Value* op1, Value* op2)
     }
 }
 
+void Llvm::buildDiv(GenTree* node, Value* op1, Value* op2)
+{
+    switch(node->TypeGet())
+    {
+        case TYP_FLOAT:
+        case TYP_DOUBLE:
+            {
+                mapGenTreeToValue(node, _builder.CreateFDiv(op1, op2));
+                break;
+            }
+        default:
+            {
+                if (varTypeIsSigned(node->TypeGet()))
+                {
+                    mapGenTreeToValue(node, _builder.CreateSDiv(op1, op2));
```

Comment:
I think it would be preferable to follow the pattern of a `Value* result;` that is initialized on the various paths and then `map`ped at the end of the method -- it makes for somewhat easier debugging.